packages/sui-mono/src/spawn.js

```diff
@@ -0,0 +1,55 @@
+/* eslint no-console:0 */
```

Comment:
YEs and this approach is more clear, documented and ready for executions of normal repos and mono repos.